Installation/Windows/client/Templates/NSIS.template.in

```diff
@@ -788,7 +788,7 @@ Function insert_registration_keys
    Return
 
  there_are_erros: ;nothing
-MessageBox MB_OK "There was an error during adding the installation keys to the registry$\nArango will work fine but there may be trouble during the deinstallation$\nplease contact @CPACK_NSIS_CONTACT@"
+MessageBox MB_OK "There was an error during adding the installation keys to the registry.$\nArangoDB will work fine, but there may be trouble during the deinstallation.$\nPlease contact @CPACK_NSIS_CONTACT@"

```

Comment:
```suggestion
MessageBox MB_OK "There was an error during adding the installation keys to the registry.$\r$\n$\r$\nArangoDB will work fine, but uninstalling it may not work properly.$\r$\n$\r$\nPlease contact @CPACK_NSIS_CONTACT@"
```